### PR TITLE
📝 added Definition of Done to standard issue template and added a PR template with the DoD

### PR DESCRIPTION
**Pull request by LuisEinig**
**Description**

**Reference**

Issues #44

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/user_story.md [hunk 1]
```diff
@@ -21,3 +21,12 @@ Als [Rolle] möchte ich [Funktionalität], damit ich [Grund].
 - [ ] Die User Story ist klar und verständlich aus der Perspektive der betroffenen Persona formuliert.
 - [ ] Vollständige und klare Akzeptanzkriterien sind definiert.
 - [ ] Die Aufwandsschätzung für das Issue wurde durchgeführt und dokumentiert.
+
+---
+**Definition of Done (DoD):**
+
+- [ ] Unittest gepflegt
+- [ ] Dokumentation ist gepflegt
```
Comment (MrSebastian) [inline .github/ISSUE_TEMPLATE/user_story.md:29]: according to the issue, these two tasks should be part of the pull request template:

```
Punkte für Pull Request:

    Unittest gepflegt
    Dokumentation ist gepflegt
```

File: .github/ISSUE_TEMPLATE/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+A short description about the request.
+
+**Reference:**
+
+Issue: #XXX
+
+---
+**Definition of Done (DoD):**
+
+- [ ] Unittest gepflegt
```
Comment (MrSebastian) [inline .github/ISSUE_TEMPLATE/pull_request_template.md:20]:
```suggestion
- [ ] Unittests gepflegt
```

**Review by MrSebastian — APPROVED** [review]

**Comment by MrSebastian** [comment]
we should update the pull request title to have a better match to changes. In the request pr-template and issue-template is getting changed.

**Review by MrSebastian — CHANGES_REQUESTED** [review]
the change from @mirrodi should be addressed

**Comment by LuisEinig** [comment]
> In order to monitor and mentor the work I would like that all the mentors are assigned as reviewers, although their approval is not needed. This should be set in the PR template

it is only possible to assign someone to the PR from the template, not to assign them as reviewers. Github does not support that functionality sadly. Maybe this can be done in the Project Settings, but I am not able to change them

**Review by Tuckerle — APPROVED** [review]

**Review by MrSebastian — APPROVED** [review]

**Pull request merged**